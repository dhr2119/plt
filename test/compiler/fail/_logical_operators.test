int main() {
  int x;
  int y;
  x = 1;
  y = 0;
  x && y;
  return 0;
}