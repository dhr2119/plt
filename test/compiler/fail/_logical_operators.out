Fatal error: exception Exceptions.InvalidBinopExpression("Can only use Bools for logical operators")
